docs/en/Configuring-Alluxio-with-OrangeFS.md

```diff
@@ -9,11 +9,20 @@ priority: 2
 This guide describes how to configure Alluxio with [OrangeFS](http://www.orangefs.org/) as the 
 under storage system.
 
-# Initial Setup
-
-First, the Alluxio binaries must be on your machine. You can either
-[compile Alluxio](Building-Alluxio-Master-Branch.html), or
-[download the binaries locally](Running-Alluxio-Locally.html).
+# Building Alluxio-OrangeFS Distro
+
+The integration of Alluxio-OrangeFS depends on 
+[orangefs-jni](http://www.orangefs.org/fisheye/orangefs/browse/orangefs/branches/maven-repository/maven2/org/orangefs/usrint/orangefs-jni) 
+for an optimised I/O performance that is comparable to OrangeFS 
+[native performance](http://docs.orangefs.com/v_2_9/Design_Overview.htm).
```

Comment:
So I did a quick test between two integration approaches, and found orangefs-jni has a better read I/O throughput. 
OrangeFS JNI (read 3,816 MB/s and write 2,903 MB/s) vs. direct mount via Kernel Module (read 2,500 MB/s and write 2,839 MB/s). I attached the benchmark details for our future reference:

**OrangeFS JNI:**
```bash
SimpleWrite Test - ID : 1460616719986
Finished Successfully
********** Task Configurations **********
files.per.thread	2
block.size.bytes	536870912
file.length.bytes	1073741824
write.type	THROUGH
dfs.perf.dfs.address	alluxio://node1248:19998
buffer.size.bytes	4194304
********** Statistics **********
SlaveName	WriteThroughput(MB/s)
0@node1248	586.0280104663631
1@node1435	632.0872131813608
2@node1893	535.7998364431123
3@node1894	570.834389496123
4@node1895	578.7144221053112
Total	2903.4638716922705
********** Slave Details **********
0@node1248's WriteThroughput(MB/s) for each threads:
	[ 151.09930647779254 ][ 144.18473669388905 ][ 145.29975168499467 ][ 145.4442156096868 ]

1@node1435's WriteThroughput(MB/s) for each threads:
	[ 159.32783569316945 ][ 155.9074299634592 ][ 151.99643758349413 ][ 164.855509941238 ]

2@node1893's WriteThroughput(MB/s) for each threads:
	[ 135.6560906140293 ][ 133.7251061051257 ][ 133.69018865461192 ][ 132.72845106934543 ]

3@node1894's WriteThroughput(MB/s) for each threads:
	[ 144.1644375615937 ][ 138.92280558947226 ][ 142.50921995685755 ][ 145.2379263881994 ]

4@node1895's WriteThroughput(MB/s) for each threads:
	[ 141.38764238867796 ][ 144.48991110483985 ][ 148.44882574659323 ][ 144.38804286520022 ]
```
```bash
SimpleRead Test - ID : 1460617069966
Finished Successfully
********** Task Configurations **********
files.per.thread	2
dfs.perf.dfs.address	alluxio://node1248:19998
read.type	CACHE_PROMOTE
buffer.size.bytes	4194304
read.mode	SEQUENCE
********** Statistics **********
SlaveName	ReadThroughput(MB/s)
0@node1248	759.9963708450913
1@node1435	759.3463451992252
2@node1893	780.4373025987757
3@node1894	741.4262662137697
4@node1895	775.1525169673024
Total	3816.3588018241644
********** Slave Details **********
0@node1248's ReadThroughput(MB/s) for each threads:
	[ 189.99907227015493 ][ 190.0872470762948 ][ 189.91097922848664 ][ 189.99907227015493 ]

1@node1435's ReadThroughput(MB/s) for each threads:
	[ 190.05196733481813 ][ 189.50680114740445 ][ 189.61207295620775 ][ 190.17550376079487 ]

2@node1893's ReadThroughput(MB/s) for each threads:
	[ 194.62130571129907 ][ 194.58432304038004 ][ 195.6064947468959 ][ 195.6251791002006 ]

3@node1894's ReadThroughput(MB/s) for each threads:
	[ 185.205281244348 ][ 184.97109826589596 ][ 185.6249433517629 ][ 185.6249433517629 ]

4@node1895's ReadThroughput(MB/s) for each threads:
	[ 193.90266994887332 ][ 193.57277882797732 ][ 193.9761318431521 ][ 193.70093634729972 ]
```

**OrangeFS Mount**
```bash
SimpleWrite Test - ID : 1460615963876
Finished Successfully
********** Task Configurations **********
files.per.thread	2
block.size.bytes	536870912
file.length.bytes	1073741824
write.type	THROUGH
dfs.perf.dfs.address	alluxio://node1248:19998
buffer.size.bytes	4194304
********** Statistics **********
SlaveName	WriteThroughput(MB/s)
0@node1248	565.6688908540486
1@node1435	513.8608827345138
2@node1893	621.1022641723262
3@node1894	611.2556893308863
4@node1895	527.1243801959665
Total	2839.012107287741
********** Slave Details **********
0@node1248's WriteThroughput(MB/s) for each threads:
	[ 157.0311301947554 ][ 148.71832110957808 ][ 149.88290398126463 ][ 110.03653556845047 ]

1@node1435's WriteThroughput(MB/s) for each threads:
	[ 103.15301702427723 ][ 160.22531685182287 ][ 142.27162209100382 ][ 108.21092676740992 ]

2@node1893's WriteThroughput(MB/s) for each threads:
	[ 155.6467548259614 ][ 154.90507525905755 ][ 155.4459203036053 ][ 155.1045137837019 ]

3@node1894's WriteThroughput(MB/s) for each threads:
	[ 150.27883768711476 ][ 160.2378530631406 ][ 153.16730237080247 ][ 147.5716962098285 ]

4@node1895's WriteThroughput(MB/s) for each threads:
	[ 110.23791581440413 ][ 146.5474060822898 ][ 160.86717461314902 ][ 109.47188368612359 ]
```
```bash
SimpleRead Test - ID : 1460616202180
Finished Successfully
********** Task Configurations **********
files.per.thread	2
dfs.perf.dfs.address	alluxio://node1248:19998
read.type	CACHE_PROMOTE
buffer.size.bytes	4194304
read.mode	SEQUENCE
********** Statistics **********
SlaveName	ReadThroughput(MB/s)
0@node1248	511.31790760277136
1@node1435	494.3290447498191
2@node1893	491.68300086626925
3@node1894	512.3548384713856
4@node1895	491.22509376282443
Total	2500.90988545307
********** Slave Details **********
0@node1248's ReadThroughput(MB/s) for each threads:
	[ 123.44786015672092 ][ 125.79852579852579 ][ 130.17224941206382 ][ 131.8992722354608 ]

1@node1435's ReadThroughput(MB/s) for each threads:
	[ 124.26430435046416 ][ 114.6889175113401 ][ 115.14027098442683 ][ 140.23555190358806 ]

2@node1893's ReadThroughput(MB/s) for each threads:
	[ 132.04384268214056 ][ 116.04056887075754 ][ 117.87728790146197 ][ 125.72130141190915 ]

3@node1894's ReadThroughput(MB/s) for each threads:
	[ 122.07188412707873 ][ 128.2565130260521 ][ 122.02097235462345 ][ 140.0054689636314 ]

4@node1895's ReadThroughput(MB/s) for each threads:
	[ 115.56909880932227 ][ 135.27974106612064 ][ 118.0199389154613 ][ 122.35631497192018 ]
```